### PR TITLE
CPO-58: case-payment-orders-api on prod

### PR DESCRIPTION
**Pull request by sullivangareth**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CPO-58

### Change description ###

flux config to add case payment orders api microservice to prod

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: k8s/namespaces/cpo/case-payment-orders-api/prod.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+kind: HelmRelease
+metadata:
+  name: case-payment-orders-api
+spec:
+  values:
+    java:
+      environment:
+        IDAM_API_URL: https://idam-api.platform.hmcts.net
+        IDAM_OIDC_URL: https://hmcts-access.service.gov.uk
+        S2S_AUTHORIZATIONS_XUI_ID: xui_webapp
```
Comment (timja) [inline k8s/namespaces/cpo/case-payment-orders-api/prod.yaml:11]: this is already set
```suggestion
```

File: k8s/namespaces/cpo/case-payment-orders-api/prod.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        S2S_AUTHORIZATIONS_XUI_PERMISSION: R
```
Comment (timja) [inline k8s/namespaces/cpo/case-payment-orders-api/prod.yaml:12]: isn't this already set as well?

**Review by timja — CHANGES_REQUESTED** [review]
cpo is missing from https://github.com/hmcts/cnp-flux-config/blob/master/k8s/prod/common-overlay/kustomization.yaml

File: k8s/namespaces/cpo/case-payment-orders-api/prod.yaml (hunk 2 above)
Comment (sullivangareth) [reply k8s/namespaces/cpo/case-payment-orders-api/prod.yaml]: Fixed in e0ae204

File: k8s/namespaces/cpo/case-payment-orders-api/prod.yaml (hunk 1 above)
Comment (sullivangareth) [reply k8s/namespaces/cpo/case-payment-orders-api/prod.yaml]: Fixed in e0ae204

**Comment by smathangi** [comment]
Please replace these entries with wildcard one as prod in place. Also please sync with latest master
https://github.com/hmcts/cnp-flux-config/blob/CPO-58/CODEOWNERS#L79-L81

**Review by smathangi — DISMISSED** [review]

**Review by danlysiak — CHANGES_REQUESTED** [review]
Looks like we're missing `identity.yaml` for prod?

**Review by danlysiak — APPROVED** [review]

**Review by cakeben — APPROVED** [review]

**Pull request merged**